services/web/client/source/class/qxapp/component/widget/NodesTree.js

```diff
@@ -159,7 +159,7 @@ qx.Class.define("qxapp.component.widget.NodesTree", {
       let data = {
         label: this.getStudyName(),
         children: this.__convertModel(topLevelNodes),
-        nodeId: "root",
+        nodeId: this.getWorkbench().getStudy().getUuid(),
```

Comment:
Leave it as was, please

Reply:
It was a request from @pcrespov

Reply:
I just have to only show it when the user is tester

Reply:
I just discussed with him. Stays as "root"

Reply:
okay!